### DISCUSSION TITLE
Q&A: Open file at the exact line

### DISCUSSION THREAD

**Discussion by zivkovicn** [ibhagwan/fzf-lua#1426] [Q&A] [answered]
Hello, recently I switched from `telescope` to `fzf-lua`, I'm loving it!

Just one thing, I'm unable to achive, some some advice fould be appreciated.

Is it possible to open file at the exact file number, let me explain:
- usually I copy paste some file path with name and file number from failing tests and that looks like this `path/fileName.go:40:21`
- if I paste this in the prompt while searching files - I don't get the results, b/c the parser doesn't know what to do with the `:`

From the screenshot below:
![Screenshot 2024-09-03 at 5 42 23 PM](https://github.com/user-attachments/assets/7815e302-cf85-459f-baaf-bb9141f890a0)
my expectation is that I can open file fzf-lua.lua at the 3rd line and position the cursor 23 character

(another screenshot just for clarification - without line number)
![Screenshot 2024-09-03 at 5 43 36 PM](https://github.com/user-attachments/assets/15db6de2-9879-4c5a-b024-278f18ffd575)

Any suggestion is appreciated. Thank you

**Comment by ibhagwan** [comment]
Duplicate of #1384.

**Comment by ibhagwan** [comment, marked as answer]
> usually I copy paste some file path with name and file number from failing tests and that looks like this `path/fileName.go:40:21`

This can only work while fuzzy matching grep results, emphasis not “live” grep, live searches only file contents and uses a regex so the `:` will only match if the line contents contains `:`.

If you want to look for matches like that you can use `:FzfLua grep search=''` (similar to `grep_ project` with fuzzy matching on the file name/line/col).

Note this will not work with `:FzfLua files` and other pickers that do not display line numbers, as there is nothing to match against the `:`.

**Reply by zivkovicn** [reply]
I see, thank you for quick answer! :)
